app/views/shared/_stream_element.mobile.haml

```diff
@@ -30,22 +30,5 @@
     != reactions_link(post)
 
     - if defined?(expanded_info) && expanded_info
-      .comment_container
-        .post_stats
-          - if post.public?
-            %span.comment_count
-              = post.reshares.size
-
-          %span.comment_count
-            = post.comments.size
-
-          %span.like_count
-            = post.likes.size
-
-        %ul.comments
-          = render :partial => 'comments/comment', :collection => post.comments.order('created_at ASC')
-
-          %li.comment.add_comment_bottom_link_container
-            - if user_signed_in? 
-              = link_to t('comments.new_comment.comment'), new_post_comment_path(post), :class => 'add_comment_bottom_link btn comment_action inactive'
+      = render :partial => 'shared/post_stats', :locals => {:post => @post}
```

Comment:
Please update to the Ruby 1.9 hash style, that we prefer as per our new [styleguide](https://wiki.diasporafoundation.org/Styleguide): `render partial: "shared/post_stats", locals: {post: @post}`